iothub/service/src/Registry/Models/JobProperties.cs

```diff
@@ -8,74 +8,15 @@
 namespace Microsoft.Azure.Devices
 {
     /// <summary>
-    /// Contains properties of a job.
+    ///
     /// </summary>
-    /// <seealso href="https://docs.microsoft.com/rest/api/iothub/service/createimportexportjob"/>
     public class JobProperties
```

Comment:
This class feels part way between a common base class and a return type. I don't think it can quite be both, TBH.